app/utils/review.js

```diff
@@ -0,0 +1,76 @@
+import { Alert, Linking } from 'react-native';
+import RNUserDefaults from 'rn-user-defaults';
+
+import { isIOS, getBundleId } from './deviceInfo';
+import I18n from '../i18n';
+import { showErrorAlert } from './info';
+
+const store = isIOS ? 'App' : 'Play';
+
+const reviewKey = 'reviewKey';
+const popupDelay = 2000;
+const numberOfDays = 7;
+const numberOfPositiveEvent = 5;
+
+// handle official and experimental app
+const id = getBundleId.includes('reactnative') ? '1272915472' : '1148741252';
```

Comment:
Can we add these links building to https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/constants/links.js?
It's easier for us to maintain and for forks.